src/platform/api/pythonApi.ts

```diff
@@ -79,7 +79,7 @@ export function pythonEnvToJupyterEnv(env: ResolvedEnvironment): PythonEnvironme
     return {
         id: env.id,
         sysPrefix: env.executable.sysPrefix,
-        envPath: Uri.file(env.path),
+        envPath: env.environment?.folderUri,
```

Comment:
This mapping was incorrect with the new API

Reply:
In the past `path` pointed to the env path, now `path` seems to point to the executable

Reply:
@karrtikr can you clarify this?

Reply:
Yup, if you check the old API, `pythonVersion.envPath` pointed to the path of the enviorinment, not the executable.
See here https://github.com/DonJayamanne/pythonVSCode/blob/main/src/client/pythonEnvironments/info/index.ts#L84

```typescript
/**
 * Details about a Python environment.
 *
 * @prop companyDisplayName - the user-facing name of the distro publisher
 * @prop displayName - the user-facing name for the environment
 * @prop type - the kind of Python environment
 * @prop envName - the environment's name, if applicable (else `envPath` is set)
 * @prop envPath - the environment's root dir, if applicable (else `envName`)
 * @prop cachedEntry - whether or not the info came from a cache
 */
// Note that "cachedEntry" is specific to the caching machinery
// and doesn't really belong here.
export type PythonEnvironment = InterpreterInformation & {
    id?: string;
    companyDisplayName?: string;
    displayName?: string;
    detailedDisplayName?: string;
    envType: EnvironmentType;
    envName?: string;
    envPath?: string;
    cachedEntry?: boolean;
};
```

Reply:
Note: its not a problem with the Python extensoin, this is basically an issue at our end, as menioend here `This mapping was incorrect with the new API`

Reply:
`env.path` is different from the former `envPath` property as Don just described, here's the doc for what it represents:

![image](https://user-images.githubusercontent.com/13199757/197951895-e19f11cf-9e56-444d-b875-c51eac79295e.png)

It's useful for the display path of the environment:

![image](https://user-images.githubusercontent.com/13199757/197952155-54a5b29e-572a-4497-b73e-d69cb5e7cd6c.png)